app/src/main/java/fr/paug/androidmakers/model/Partners.kt

```diff
@@ -0,0 +1,12 @@
+package fr.paug.androidmakers.model
+
+class Partners(val name: String?, val imageUrl: String?, val link: String?, val description: String?) {
```

Comment:
Would it be better to go for a data class? :)

Reply:
👍

Reply:
I did it in the next PR to avoid conflicts when moving files: https://github.com/paug/AndroidMakersApp/commit/68075dc991dbfab5d27ef68f321aeb1c9051bee5

For the record, I still think something like https://github.com/drewhamilton/Poko (or even thinner) would be a better fit because `data classes` come with  a lot of extra byte-code that we don't really use.